crypto/keys/ed25519/ed25519.go

```diff
@@ -1,14 +1,14 @@
 package ed25519
 
 import (
+	"crypto/ed25519"
```

Comment:
This is good BTW

Reply:
yes, I made a note in the commit. Here are the detail: https://golang.org/doc/go1.13#crypto/ed25519
We shouldn't use golang.org/x/crypto/ed25519 package.